docs/src/test/java/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -91,9 +91,13 @@ public ExamplePerson parse(ByteString line) {
     final CompletionStage<HttpEntity.Strict> strictEntity = response.entity()
         .toStrict(FiniteDuration.create(3, TimeUnit.SECONDS).toMillis(), system);
```

Comment:
Maybe while add it, also add explicit max number of bytes here or add a comment how the maximum is determined otherwise.